docs/Phidget.md

```diff
@@ -10,7 +10,8 @@ The PhidgetsAPI package exposes a few different ways of interacting with your Ph
 |phidget.connect|[__phidget__.params object](#connecting--phidgetparams)|This is the main initialize function.  Params is a JSON array of connection variables.  The phidgetReady event will be dispatched upon connection and initialization success. You may wish to bind other listeners to your __phidget__ inside a listener for this event.|
 |phidget.set|[__phidget__.set object](#setting-information-phidgetset)|This method is used to set any output or setable device ( onboard led etc ) on your Phidget. See your __phidget__.data object for possible outputs. Remember this is case sensative so match that case exactly as it is in the __phidget__.data object for your device.
 |phidget.quit|N/A |This method requests a disconnect from the phidget board.  The disconnected event will be dispatched when the connection has been successfully disconnected. |
-|phidget.on|"event name", eventHandler| This is how you bind to the phidget devices events.|
+|phidget.on, phidget.addListener|"event name", eventHandler| This is how you bind to the phidget devices events.|
+|phidget.removeListener|"event name", eventHandler| This is how you release any bound events to the phidget devices events.|
```

Comment:
I would say how you release or ` remove ` ( to keep terminology same) a specific event handler. It doesn't remove all/any event handlers, it only releases the specific handler function reference you pass it.